docs/source/getting-started/examples_intro.rst

```diff
@@ -0,0 +1,85 @@
+     - Number of Hours Per Week Usually Worked
+     - 00: Usual hours cannot be given because hours worked vary considerably from week to week or from month to month \n 01-98: Number of hours usually worked in the main job \n 99: Not applicable (WSTATOR=3-5,9) \n blank: No answer
+   * - `QUARTER`
+     - Fixed Reference Quarter
+     - Single Quarter
+   * - `YEAR`
+     - Fixed Reference Year
+     - Single Year
+
+Compositor Overview
+-------------------
+The compositor is the foundation of our differentially private queries. It essentially takes in our data and our specifications for the queries that we would like to run. At this point, we won't be directly referencing our data again and we could theoretically delete it! 
+
+.. code-block:: python
+
+   context = dp.Context.compositor(
+       data=df,
+       privacy_unit=dp.unit_of(contributions=36),
+       privacy_loss=dp.loss_of(epsilon=1.0),
+       split_evenly_over=10,
+       margins={
+           ("SEX", ): dp.Margin(max_partition_length=60_000_000),
+           ("AGE", ): dp.Margin(max_partition_length=60_000_000),
+           ("ILOSTAT", ): dp.Margin(max_partition_length=60_000_000),
+           ("HWUSUAL", ): dp.Margin(max_partition_length=60_000_000),
+           ("YEAR", ): dp.Margin(max_partition_length=60_000_000, max_partition_contributions=4),
+           ("QUARTER", ): dp.Margin(max_partition_length=60_000_000, max_partition_contributions=13),
+           ("YEAR", "QUARTER",): dp.Margin(max_partition_length=60_000_000, max_partition_contributions=1),
+           (): dp.Margin(max_partition_length=60_000_000),
+       },
+   )
+
+**Parameters**:
+
+* *privacy_unit:* How many rows each individual or entity of interest contributes to our data frame. In this case, we are analyzing the data from the first quarter of 2008 and each individual is represented once. Hence, the privacy unit is 1. If we were to analyze the dataset across quarters and years, each individual would contribute (number of quarters * number of years) times.
+
+* *privacy_loss:* This parameter determines how much privacy we want to preserve. If ε is small, we will have more privacy but worse data accuracy. ε can range from 0 to infinity, but 1 is usually a standard. For now, we won't specify an ε.
+
+* *split_evenly_over:* This is the number of queries you want to distribute your privacy loss over.
+
+* *margins:* Margins capture the variables of interest in your analysis. This can include variables that you may want to group by or apply differential privacy techniques to. 
+    * *max_partition_length:* The upper bound on how many records (individuals in this case) can be in one partition. This is generally an upper bound on the population represented in your dataset if you do not know the size of your dataset. In this case, the population of France was about 60 million in 2004 so that's our maximum partition length. Source: `World Bank <https://datatopics.worldbank.org/world-development-indicators/>`. 
```

Comment:
```suggestion
    * *max_partition_length:* The upper bound on how many records (individuals in this case) can be in one partition. If you do not know the size of your dataset, this can be an upper bound on the population represented in your dataset. The population of France was about 60 million in 2004 so that's our maximum partition length. Source: `World Bank <https://datatopics.worldbank.org/world-development-indicators/>`_. 
```